File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseCaducidad.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.CaducidadDTO;

/**
 * Objeto de respuesta con la informacion de la caducidad
 * 
 * @author Flor
 *
 */
public class ServerResponseCaducidad {

	private String idCaducidad = null;

	private List<CaducidadDTO> listaCaducidad = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseCaducidad() {
		super();
	}

	public String getIdCaducidad() {
		return idCaducidad;
	}

	public void setIdCaducidad(String idCaducidad) {
		this.idCaducidad = idCaducidad;
	}

	public List<CaducidadDTO> getListaCaducidad() {
		return listaCaducidad;
	}

	public void setListaCaducidad(List<CaducidadDTO> listaCaducidad) {
		this.listaCaducidad = listaCaducidad;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseCaducidad.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseCaducidad
    {
        private string _idCaducidad;

        public string idCaducidad
        {
            get { return _idCaducidad; }
            set { _idCaducidad = value; }
        }

        private List<CaducidadDTO> _listaCaducidad;

        public List<CaducidadDTO> listaCaducidad
        {
            get { return _listaCaducidad; }
            set { _listaCaducidad = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
